Community board Q4 blog post (#2060)
writethedocs/www: docs/blog/2023-Q4-community-board.rst

.. post:: Dec 1, 2023
   :tags: community board, news, report
   :author: Mikey Ariel

Write the Docs Community Board Quarterly Update Q4-2023
=======================================================

Greetings, documentarians! Mikey Ariel here, chairperson of the WTD community board. I'm happy to share with you the fourth and final quarterly update for 2023. 

To catch up on the previous quarterly updates, check out the `community board blog posts <https://www.writethedocs.org/blog/archive/tag/community-board/>`_ archive. 

Q4 quarterly meeting highlights
-------------------------------

Our Q4 meeting covered multiple new topics and follow-ups on previously discussed topics. Here are some of the highlights: 

**Slack moderation in times of increased spamming**

>     Our Slack space has always been safe and friendly to our community members. However, recently there have been more reports to the moderation team about "spam" posts. While not necessarily malicious, these are mostly self-promotion links to blog posts and articles without context and off-topic for the channels where they're posted. The Slack moderation team has been diligent and thorough in handling these reports, and they will continue to monitor the channels alongside our dedicated community memebers who help with these reports.

Author: ```suggestion
    Our Slack space has always been safe and friendly to our community members. However, recently there have been more reports to the moderation team about "spam" posts. While not necessarily malicious, these are mostly self-promotion links to blog posts and articles without context and off-topic for the channels where they're posted. The Slack moderation team has been diligent and thorough in handling these reports, and they will continue to monitor the channels alongside our dedicated community members who help with these reports.
```

**Meetup organizer mailing list and survey**

>     In Q3, the Rose from the Meetups team launched a survey for all registered meetup organizers in order to get the most up-to-date state of their meetups. The results of the survey are still coming in, but so far she was able to determine that multiple meetups were not active and did not have plans to resume their activities in the near future. These meetups will be offloaded from the WTD subscription plan, and if they decide to resume activities then they will be able to restart their subscription. 

Reviewer: ```suggestion
    In Q3, Rose from the Meetups team launched a survey for all registered meetup organizers in order to get the most up-to-date state of their meetups. The results of the survey are still coming in, but so far she was able to determine that multiple meetups were not active and did not have plans to resume their activities in the near future. These meetups will be offloaded from the WTD subscription plan, and if they decide to resume activities then they will be able to restart their subscription. 
```

> **Meetup organizer mailing list and survey**
>     In Q3, the Rose from the Meetups team launched a survey for all registered meetup organizers in order to get the most up-to-date state of their meetups. The results of the survey are still coming in, but so far she was able to determine that multiple meetups were not active and did not have plans to resume their activities in the near future. These meetups will be offloaded from the WTD subscription plan, and if they decide to resume activities then they will be able to restart their subscription. 

Reviewer 2: ```suggestion
**Meetup organizer mailing list and survey**
    In Q3, Rose from the Meetups team launched a meetup organizer survey to find out which meetups were still active. The results of the survey are still coming in, but so far she was able to determine that multiple meetups were not active and did not have plans to resume their activities in the near future. These meetups will be offboaded from the WTD subscription plan, and if they decide to resume activities then they will be able to restart their subscription. 
```

    
    Several other meetups confirmed that they are planning to or in the process of resuming their activities. To help the meetups coordinate their events and learn about other regions' meetups, the meetup organizers were invited to subscribe to a new shared calendar where all the meetups can be listed.
    

>     Once again, we appreciate all of the hard work that meetup organizers do to keep their communities engaged, and will continue to work with the meetup teams to adapt to the current the global landscape. If you want to start a new meetup or join an existing meetup, check out the `Meetups page <https://www.writethedocs.org/meetups/>`_ on our website.

Reviewer: ```suggestion
    Once again, we appreciate all of the hard work that meetup organizers do to keep their communities engaged, and will continue to work with the meetup teams to adapt to the current global landscape. If you want to start a new meetup or join an existing meetup, check out the `Meetups page <https://www.writethedocs.org/meetups/>`_ on our website.
```

**2023 salary survey promotion**

>     Now that the `2022 salary survey <https://www.writethedocs.org/surveys/salary-survey/2022/>`_ has been open for a little while, we are starting to see the data that is coming in as well as feedback. We are working on social media prompts to remind the community to participate before the survey closes at the end of this year, and special prompts for contractors and freelancers that will help us get enough responses to the survey. 

Author: ```suggestion
    The `2023 salary survey <https://salary-survey.writethedocs.org/>`_ officially launched in September, and we are already starting to see great data coming in as well as valuable feedback from the participants. We are working on social media prompts to remind the community to participate before the survey closes at the end of this year, and special prompts for contractors and freelancers that will help us get enough responses to the survey. 
```

>     The `2023 salary survey <https://salary-survey.writethedocs.org/>`_ officially launched in September, and we are already starting to see great data coming in as well as valuable feedback from the participants. We are working on social media prompts to remind the community to participate before the survey closes at the end of this year, and special prompts for contractors and freelancers that will help us get enough responses to the survey. 

Reviewer 2: ```suggestion
    The `2023 salary survey <https://salary-survey.writethedocs.org/>`_ officially launched in September, and we are seeing lots of responses coming in, as well as valuable feedback from the participants. We are working on social media prompts to remind the community to participate before the survey closes at the end of this year, and special prompts for contractors and freelancers that will help us get enough responses to the survey. 
```

This feels too detailed, but :man_shrugging:

> 

Reviewer 2: @goatlady anything to add?

Author: I need to merge this today in order to make the newsletter, but I can always tweak post-release if needed
